composer.json

```diff
@@ -33,9 +33,9 @@
     "description" : "Codeception extension for Phiremock. Allows to stub remote services for HTTP requests.",
     "license" : "GPL-3.0-or-later",
     "require" : {
-        "php" : "^8.0",
+        "php" : "^8.2",
```

Comment:
Is it possible to have symfony process required as `">=5.0.0 <8.0.0"` and keep the php "^8.0" requirement?
This change will require me to release a major version of the extension (v4), and then I would need to maintain v3 and v4, since probably there are still people using php v8.0 and v8.1.